### PR TITLE
#73 Completed

### PR DESCRIPTION
**Pull request by Bekmurod21**
Problem #77  has been solved

### CODE REVIEW

**Review by FindAbdulloh — APPROVED** [review]

File: src/FleetFlow.Api/Middlewares/ExceptionHandlerMiddleware.cs [hunk 1]
```diff
@@ -29,6 +31,7 @@ public async Task Invoke(HttpContext context)
             }
             catch(Exception exception)
             {
+                this.logger.LogError(exception.ToString());
```
Comment (kerakmas) [inline src/FleetFlow.Api/Middlewares/ExceptionHandlerMiddleware.cs:34]: while you are writing to file you need to take some space in order to divide to exeptions. you can paste this code below
 this.logger.LogError($"{exception.ToString()}\n");

**Review by kerakmas — COMMENTED** [review]
correct  34th line in  ExceptionHandlerMiddleware.cs

File: src/FleetFlow.Api/Middlewares/ExceptionHandlerMiddleware.cs (hunk 1 above)
Comment (Bekmurod21) [reply src/FleetFlow.Api/Middlewares/ExceptionHandlerMiddleware.cs]: I corrected it

Comment (kerakmas) [reply src/FleetFlow.Api/Middlewares/ExceptionHandlerMiddleware.cs]: > I corrected it
Nice

**Review by kerakmas — APPROVED** [review]

**Review by MuzaffarNurillaew — APPROVED** [review]
Everything is ok, configurations has been added to appsettings and Program.cs. And implemented in ExceptionHandlerMiddleware

**Pull request merged**